groups.go

```diff
@@ -29,15 +34,96 @@ func (h *groupsHandler) Create(c *gin.Context) {
 }
 
 func (h *groupsHandler) Delete(c *gin.Context) {
+	bearer, err := h.authenticate(c)
+	if err != nil {
+		writeError(c, http.StatusUnauthorized, err)
+		return
+	}
+
 	body := &accountsv1.DeleteGroupRequest{
 		Id: c.Param("id"),
 	}
 
-	res, err := h.groupsClient.DeleteGroup(context.Background(), body)
+	res, err := h.groupsClient.DeleteGroup(contextWithGrpcBearer(context.Background(), bearer), body)
+	if err != nil {
+		writeError(c, http.StatusInternalServerError, err)
+		return
+	}
+
+	c.JSON(http.StatusOK, res)
+}
+
+func (h *groupsHandler) ListMembers(c *gin.Context) {
+	bearer, err := h.authenticate(c)
+	if err != nil {
+		writeError(c, http.StatusUnauthorized, err)
+		return
+	}
+
+	body := &accountsv1.ListGroupMembersRequest{}
+	if err := c.ShouldBindJSON(body); err != nil {
+		c.JSON(http.StatusOK, httpError{Error: err.Error()})
+		return
+	}
+
+	body.Id = c.Param("id")
+
+	res, err := h.groupsClient.ListGroupMembers(contextWithGrpcBearer(context.Background(), bearer), body)
 	if err != nil {
 		writeError(c, http.StatusInternalServerError, err)
 		return
 	}
 
 	c.JSON(http.StatusOK, res)
 }
+
+func (h *groupsHandler) Update(c *gin.Context) {
+	bearer, err := h.authenticate(c)
+	if err != nil {
+		writeError(c, http.StatusUnauthorized, err)
+		return
+	}
+
+	body := &accountsv1.UpdateGroupRequest{}
+	if err := c.ShouldBindJSON(body); err != nil {
+		c.JSON(http.StatusOK, httpError{Error: err.Error()})
+		return
+	}
+	body.Group.Id = c.Param("id")
+	res, err := h.groupsClient.UpdateGroup(contextWithGrpcBearer(context.Background(), bearer), body)
+	if err != nil {
+		writeError(c, http.StatusInternalServerError, err)
+		return
+	}
+
+	c.JSON(http.StatusOK, res)
+}
+
+func (h *groupsHandler) Join(c *gin.Context) {
+	bearer, err := h.authenticate(c)
+	if err != nil {
+		writeError(c, http.StatusUnauthorized, err)
+		return
+	}
+
+	body := &accountsv1.JoinGroupRequest{}
+	body.Id = c.Param("id")
```

Comment:
Not very important but:
```suggestion
	body := &accountsv1.JoinGroupRequest{
		Id: c.Param("id")
	}
```